proto/tendermint/abci/types.proto

```diff
@@ -275,17 +269,16 @@ message LastCommitInfo {
 // Later, transactions may be queried using these events.
 message Event {
   string                  type       = 1;
-  repeated EventAttribute attributes = 2 [
-    (gogoproto.nullable) = false,
-    (gogoproto.jsontag)  = "attributes,omitempty"
-  ];
+  repeated EventAttribute attributes = 2 [(gogoproto.nullable) = false, (gogoproto.jsontag) = "attributes,omitempty"];
 }
 
 // EventAttribute is a single key-value pair, associated with an event.
 message EventAttribute {
-  bytes key   = 1;
-  bytes value = 2;
-  bool  index = 3;  // nondeterministic
+  // bytes key   = 1;
+  // bytes value = 2;
+  string key   = 1;
+  string value = 2;
+  bool   index = 3;  // nondeterministic
```

Comment:
@ethanfrey should we break proto here by changing the types or should we introduce new types, `key_str` and `value_str`, and use those while keeping the existing ones deprecated?

/cc @tychoish

Reply:
Good question. Responded to this in the issue with my opinion. Curious as to the implications of the queries. Or binary.

If bytes field held utf8 string before, then these two are identical. It only breaks when illegal binary data was present

Reply:
Yeah I think this change is fine. I'll wrap this up then.